ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/loopback-interfaces.j2

```diff
@@ -9,27 +9,30 @@
 | Interface | Description | VRF | IP Address |
 | --------- | ----------- | --- | ---------- |
 {%     for loopback_interface in loopback_interfaces | arista.avd.natural_sort %}
-| {{ loopback_interface }} | {{ loopback_interfaces[loopback_interface].description | arista.avd.default('-') }} |
-{%-        if loopback_interfaces[loopback_interface].vrf is defined %} {{ loopback_interfaces[loopback_interface].vrf }} {%         else %} default {%         endif -%} |
-{%-        if loopback_interfaces[loopback_interface].ip_address is defined %}
- {{ loopback_interfaces[loopback_interface].ip_address }}
-{%-            if loopback_interfaces[loopback_interface].ip_address_secondaries is defined and loopback_interfaces[loopback_interface].ip_address_secondaries is not none %}
- <br> {{ loopback_interfaces[loopback_interface].ip_address_secondaries | join(' secondary <br> ') }} secondary
-{%-            endif %}
-{%-        else %}
- -
-{%-        endif %}
- |
+{%         if loopback_interfaces[loopback_interface].vrf is defined %}
```

Comment:
This can be refactored to `set vrf = loopback_interfaces[loopback_interface].vrf | arista.avd.default('default')` and then remove line 13,14 and the default filter in line 21.

Reply:
Done

Reply:
this is not working , by this it is showing variable vrf undefined.

Reply:
adding default filter in line 21 is working fine.